modules/ROOT/pages/anypoint-mq/anypoint-mq-connector-reference.adoc

```diff
@@ -323,7 +337,7 @@ Anypoint MQ Subscriber Message Source, retrieves messages from the given destina
 [%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
```

Comment:
Again the "a" in the column title. This might be intentional. If so, disregard. There are some more of these, but I don't want to keep pointing it out in case it's supposed to be there. So just keep an eye out if the "a" should not be part of the column title.